.github/workflows/check-prerequisites.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Woohoo, more GitHub PR checks 😄

Reply:
Split off to here https://github.com/microsoft/azurelinux/pull/11081